tests/x11regressions/gedit/gedit_save.pm

```diff
@@ -43,13 +43,17 @@ sub run() {
     sleep 1;
 
     # save and quit
-    send_key "ctrl-s", 1;
+    wait_screen_change {
+        send_key "ctrl-s";
+    };
     send_key "ctrl-q";
 
     # open saved file to validate
     x11_start_program("gedit " . "test.txt");
     assert_screen 'gedit-saved-file', 3;
-    send_key "ctrl-q",                1;
+    wait_screen_change {
+        send_key "ctrl-q";
+    }
```

Comment:
;

maybe it looks safer it we write this special construct in one line
```
wait_screen_change { send_key "ctrl-q"; };
```
or even
```
wait_screen_change { send_key "ctrl-q" };
```